src/tests/ftest/scripts/main.sh

```diff
@@ -179,13 +179,47 @@ if [ "$(lsb_release -s -i)" = "CentOS" ]; then
 else
     process_cores=""
 fi
+
+# Clean stale job results
+if [ -d "$logs_prefix/ftest/avocado/job-results" ]; then
+    rm -rf "$logs_prefix/ftest/avocado/job-results"
+fi
+
 # now run it!
 # shellcheck disable=SC2086
-if ! ./launch.py -cris"${process_cores}"a -th "${LOGS_THRESHOLD}" \
+if ! ./launch.py -jcris"${process_cores}"a -th "${LOGS_THRESHOLD}" \
                  -ts "${TEST_NODES}" ${NVME_ARG} ${TEST_TAG_ARR[*]}; then
     rc=${PIPESTATUS[0]}
 else
     rc=0
 fi
 
+# daos_test uses cmocka framework which generates a set of xml of its own
+# post-processing the xml files here to put them in proper categories
+# for publishing in Jenkins
+FILES="$logs_prefix/ftest/avocado/job-results/daos_test/*"
+echo $(ls $FILES)
```

Comment:
(lint) Quote this to prevent word splitting. [SC2046]
(lint) Useless echo? Instead of 'echo $(cmd)', just use 'cmd'. [SC2005]
(lint) Double quote to prevent globbing and word splitting. [SC2086]

Reply:
All of the shellcheck warnings should be fixed.  In particular, `echo $(cmd)` should just be `$cmd`.